include/rocksdb/c.h

```diff
@@ -419,6 +419,10 @@ extern ROCKSDB_LIBRARY_API void rocksdb_drop_column_family(
 extern ROCKSDB_LIBRARY_API void rocksdb_column_family_handle_destroy(
     rocksdb_column_family_handle_t*);
 
+extern ROCKSDB_LIBRARY_API void rocksdb_list_column_family_options_destroy(
+    char** list_column_family_names,
+    rocksdb_options_t** list_column_family_options, size_t len);
```

Comment:
I sort of feel this should be a higher-level API (`rocksdb_load_latest_options_destroy()`) and free everything allocated by `rocksdb_load_latest_options()` including the `db_options`. Or, it could  be omitted entirely and let users call the low-level APIs. Right now it's kind of in-between and therefore not obvious for users of `rocksdb_load_latest_options()` how to find it unless they read the test case.

Reply:
Thanks for pointing it out.  I like the `rocksdb_load_latest_options_destroy()` idea, and it's more consistent with the existing C api convention as well.  Let me give it a try.